lib/manager.js

```diff
@@ -243,6 +243,18 @@ Manager.prototype.close = function (force, fn) {
 }
 
 /**
+ * Lists all collections.
+ *
+ * @param {Object} [query] - A query expression to filter the list of collections.
+ * @return {Array} array of all collections
+ */
+
+Manager.prototype.listCollections = function (query) {
+  if (!query) query = undefined
+  return this._db.listCollections(query).toArray()
```

Comment:
pretty sure you need to use `this.executeWhenOpened().then(function (db) {...})` so that it's only executed when the connection is opened